lib/src/clojure_lsp/feature/workspace_symbols.clj

```diff
@@ -27,33 +27,25 @@
 (defn ^:private element->workspace-symbol [element]
   {:name (f.document-symbol/element->name element)
    :kind (f.document-symbol/element->symbol-kind element)
-   :location {:filename (:filename element) ;; will be replaced with uri
+   :location {:uri (:uri element)
               :range (shared/->scope-range element)}})
 
-(defn ^:private symbol-with-uri [symb uri]
-  (-> symb
-      (update :location dissoc :filename)
-      (update :location assoc :uri uri)))
-
 (defn workspace-symbols [query db]
   ;; TODO refactor to be a complete transducer
   (->> (q/find-internal-definitions db)
        (fuzzy-filter query)
        (map element->workspace-symbol)
-       ;; Group elements by file, but otherwise preserve ordering by search score.
-       ;; Also replace filename with uri.
-       (reduce (fn [{:keys [next-idx filenames] :as result} symb]
-                 (let [filename (:filename (:location symb))]
-                   (if-let [{:keys [idx uri]} (get filenames filename)]
-                     (update-in result [:members idx] conj (symbol-with-uri symb uri))
-                     (let [uri (shared/filename->uri filename db)]
```

Comment:
Removes some code that let us limit the number of times we converted a particular filename to a URI.